### DISCUSSION TITLE
How to make store/update of resource page under transaction?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#6144] [Help]
Using  Repeater in laravel 9 / filamentphp 2  app I can make sevaral operations in one resource page(in store/update modes )
and checking generated sql statements I see several  sql statements which are not under 1 transaction.

That seems not good, can I set transaction in this case ? Or in commno having some big form with a lot of related data...

Thanks!

**Comment by danharrin** [comment]
We don't use transactions yet, you could PR it into our code if you want or maybe try using a lifecycle hook.

**Reply by sergeynilov** [reply]
Could, you please to add  transactions support into the project ?

**Reply by danharrin** [reply]
I have asked you to PR it, I am not going to do this myself as I have other feature priorities but I will merge a PR if you do the work

**Comment by kedniko** [comment]
What is the status of this discussion?
